Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -84,6 +84,38 @@ open class ChatMessageListVC: _ViewController,
         return (!listView.isLastCellFullyVisible && isMoreContentThanOnePage) || dataSource?.isFirstPageLoaded == false
     }
 
+    /// A button to scroll the collection view to the first unread message.
+    /// Visible when there are unread messages outside of the bounds of the screen.
+    open private(set) lazy var jumpToUnreadMessagesButton: JumpToUnreadMessagesButton = components
+        .jumpToUnreadMessagesButton
+        .init()
+        .withoutAutoresizingMaskConstraints
+
+    /// A Boolean value indicating whether jump to unread messages button is visible.
+    open var showJumpToUnreadMessages: Bool {
+        guard let dataSource = dataSource,
+              dataSource.sessionUnreadCount.messages > 0 else {
```

Comment:
I wonder if we can move this `sessionUnreadCount` to the LLC, or if we can't somehow reuse the `channel.unreadCount`?

Reply:
Same as before, the logic is UI only, and it does not make sense to be part of the LLC